globals.go

```diff
@@ -674,9 +679,9 @@ var (
 	}
 
 	// constant arrays for db update logic
-	AbortableStatuses   = []string{TaskStarted, TaskDispatched}
+	AbortableStatuses   = []string{TaskStarted, TaskDispatched, TaskContainerAllocated}
```

Comment:
My understanding of aborting a task is that a task isn't aborted until it's actually dispatched to an agent. I assume that we should keep the existing behavior of being able to unschedule until the task is dispatched, and only then allowing the task to be aborted.